rmf_demo_panel/task_list/airport_tasks.json

```diff
@@ -0,0 +1 @@
+[{"task_type":"Delivery", "start_time":0, "description": {"option": "mop"}}]
```

Comment:
Let's delete the `rmf_demo_panel/task_list/` folder. The contents have already been moved here https://github.com/osrf/rmf_demos/tree/develop/dispatcher_demo/rmf_demo_tasks/rmf_demo_tasks